docs/swagger_v2.yml

```diff
@@ -216,6 +216,10 @@ paths:
           description: Function definition
           schema:
             $ref: '#/definitions/Fn'
+        404:
```

Comment:
You don't really need these, the default handles all of this. I removed them earlier.